### PR TITLE
Update courses in db to summer 2021

### LINKED ISSUE #153: Schedule Scrape/Locally Update Courses (Summer 2021)
Can update courses now that the summer schedule is up.

### PR DESCRIPTION
**Pull request by johnagl**
# Changes
- Updated courses to summer 2021 locally
- Added `page.setDefaultNavigationTimeout(0);` to prevent timeouts when scraping

## Motivation and Context
Courses are outdated in production

## How Has This Been Tested?
Locally

## Screenshots (if appropriate):

# Related Issues
- Closes issue #153 

# Checklist

### CODE REVIEW

File: server/utils/scraper.ts [hunk 1]
```diff
@@ -269,6 +269,7 @@ let formatSectionInfo = async (elmHandle: ElementHandle, browser: Browser): Prom
  * @param {Page} page The Page to be configured
  */
 let pageConfig = async (page: Page) => {
+    await page.setDefaultNavigationTimeout(0);
```
Comment (yehee) [inline server/utils/scraper.ts:272]: What happens if it times out? Would we get any notifications or do we just do a quick sanity check after?

Comment (johnagl) [reply server/utils/scraper.ts:272]: So... I think some of the pages on SSC take a long time to scrape. Before I added this line, it kept timing out after 30000ms (which is default). This just lets the scraper keep going - and yes we can do a sanity check locally. 

Also, there are a few additional issues - which will probably force us to do manual scrapes instead of scheduled ones for now. Firstly, after testing it out a bit, scheduled scrapes with specific dates and times don't seem to work. Only certain types of scheduled scrapes work like , scrape every 2 minutes. I think it's been reported in node-cron https://github.com/node-cron/node-cron/issues/261 recently. In fact, we should probably get comment out the scheduled scraper for now in part because of the next problem.

Secondly, our db setup currently uses courses from the output json file. I realized that even if the scheduled scraper were working, I don't think our output json file actually gets updated (ie: heroku can't push changes to our git repo and trigger a build) and I don't think Heroku can update the code base by itself (@resurreccionl can you confirm if this logic is correct?). Basically, the fix would be to edit our scraper to insert rows into the db directly instead of adding it to an output json - but we probably don't have time for that right now.

In short:
1) `await page.setDefaultNavigationTimeout(0)` fixes local time outs when scraping (the default of 30k ms is too short) - sanity check probably necessary 
2) node-cron seems to be broken for (some) scheduled scrapes
3) Our db setup relies on output.json changing (this is a code change), I don't think Heroku can update our code base so we need to do local scrapes for now to update course info.

I commented out the scheduled scrapes for now.

Comment (resurl) [reply server/utils/scraper.ts:272]: yikes, i wasn't aware of it until you mentioned the possibility, but yes heroku doesn't persist changes to the filesystem because of how constantly dynos restart 
https://devcenter.heroku.com/articles/dynos#restarting
https://devcenter.heroku.com/articles/dynos#ephemeral-filesystem
pretty disappointing that `node-cron` stopped working consistently too

Comment (johnagl) [reply server/utils/scraper.ts:272]: Yeah I didn't realize until now. It's okay though since manual scrapes are easy enough and node-cron should get fixed soon
